packages/domain-search/src/components/domain-search-controls/input.tsx

```diff
@@ -0,0 +1,46 @@
+import { SearchControl } from '@wordpress/components';
+import './input.scss';
+
+export const DomainSearchControlsInput = ( {
+	value,
+	label,
+	onChange,
+	onReset,
+	autoFocus,
+	onBlur,
+	minLength,
+	maxLength,
+	dir,
+	'aria-describedby': ariaDescribedBy,
+}: {
+	value: string;
+	label: string;
+	onChange: ( value: string ) => void;
+	onReset: () => void;
+	autoFocus: boolean;
+	onBlur: ( event: React.FocusEvent< HTMLInputElement > ) => void;
+	minLength: number;
+	maxLength: number;
+	dir: 'ltr' | 'rtl';
+	'aria-describedby': string;
+} ) => {
+	return (
+		<SearchControl
+			className="domain-search-controls__input"
+			__nextHasNoMarginBottom
+			hideLabelFromVision
+			placeholder="Search..."
```

Comment:
We need to i18n this, and also use `…` instead of three single dots.

Use i18n like this: https://github.com/Automattic/wp-calypso/blob/add/domain-search-control-input/packages/domain-search/src/components/domain-suggestion-cta/index.tsx#L42